ADDRESSING.md

```diff
@@ -190,7 +187,7 @@ A [`suborigin` header](https://w3c.github.io/webappsec-suborigins/#the-suborigin
 SHOULD be returned by HTTP gateway and contain a value
 unique to the current content addressing root.
 
-Unfortunately due to limited adoption suborigin have no practical use.
+Unfortunately due to limited adoption suborigin has no practical use and is considered abandoned.
```

Comment:
maybe worth laying that at the feet of browser/w3c and linking to a relevant decision.